src/styles/index.css

```
@@ -19,104 +19,106 @@
 /* Light mode color values - default */
 @layer base {
   :root {
-    --color-background: theme(--color-white);
-    --color-foreground: theme(--color-zinc-950);
-    --color-card: theme(--color-white);
-    --color-card-foreground: theme(--color-zinc-950);
-    --color-primary: theme(--color-purple-600);
+    /* Glinda — Light Mode: Pinks only */
+    --color-background: theme(--color-pink-100);
```

Biome (2.1.2)

[error] Unexpected unknown function: theme

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)